src/logger.js

```diff
@@ -305,6 +308,12 @@ class Logger extends Writable {
         modifiedLog[safeLevel] = lvlName;
       }
 
+      //  Set level if present
+      if (this.withHostname) {
+        safeHost = getSafeProp(modifiedLog, 'host');
```

Comment:
might be nice to keep the naming the same everywhere
```suggestion
        safeHost = getSafeProp(modifiedLog, 'hostname');
```